.travis.yml

```diff
@@ -112,6 +112,14 @@ jobs:
       script: ./tools/travis_controller.sh tests
       env: PROFILE="-Dhadoop.version=2.8.3 -Dinclude_hadoop_aws -Dscala-2.11"
       name: tests
+    - if: type in (pull_request, push)
+      script: ./tools/travis_controller.sh scheduler_ng_core
+      env: PROFILE="-Dhadoop.version=2.8.3 -Dinclude_hadoop_aws -Dscala-2.11"
+      name: scheduler_ng_core
```

Comment:
let's follow the commong parrtn `$stage - $modifies`, i.e. `core - scheduler_ng`

Reply:
done.